os/kernel/semaphore/sem_holder.c

```diff
@@ -793,6 +793,7 @@ void sem_destroyholder(FAR sem_t *sem)
 	}
 
 	sem->holder.htcb = NULL;
+	sem->holder.counts = 0;
```

Comment:
I don't get this commit. holder->counts is being initialized to zero at allocation time. Is there any point in resetting it here? I do not know what this commit means.